packages/valory/skills/decision_maker_abci/behaviours/base.py

```diff
@@ -609,6 +614,7 @@ def _write_benchmark_results(
         p_no: Optional[float] = None,
         confidence: Optional[float] = None,
         bet_amount: Optional[float] = None,
+        liquidity_info: Optional[LiquidityInfo] = None,
```

Comment:
```suggestion
        liquidity_info: LiquidityInfo,
```

Reply:
We cannot declare it like this without a default value, because the previous parameters have default values.